modules/op-creating-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -10,273 +10,16 @@ To run pipelines using {pac}, you can create pipeline run definitions or templat
-
-:FeatureName: Using the header and body of the payload for CEL-based filtering with {pac}
-include::snippets/technology-preview.adoc[]
-
-In the following example, the pipeline run starts only if all of the following conditions are true:
-
-* The pull request is targeting the `main` branch.
-* The author of the pull request is `superuser`.
-* The action is `synchronize`; this action triggers when an update occurs on a pull request.
-
-[source,yaml]
-----
-apiVersion: tekton.dev/v1
-kind: PipelineRun
-metadata:
-  annotations:
-    pipelinesascode.tekton.dev/on-cel-expression: |
-      body.pull_request.base.ref == "main" &&
-        body.pull_request.user.login == "superuser" &&
-        body.action == "synchronize"
-# ...
-----
-
-[NOTE]
-====
-If you use the `header` or `body` field for event matching, you might be unable to trigger the pipeline run using Git commands such as `retest`. If you use a Git command, the payload body is the comment that contains this command and not the original payload.
-
-If you want to trigger the pipeline run again when using the `body` field for event matching, you can close and reopen the pull request or merge request, or alternatively add a new SHA commit, for example using the following command:
-
-[source,terminal]
-----
-git commit --amend --no-edit && git push --force-with-lease
-----
-====
-
-[discrete]
-.Using the temporary GitHub App token for Github API operations
-
-You can use the temporary installation token generated by {pac} from GitHub App to access the GitHub API. The token value is stored in the temporary `{{git_auth_secret}}` dynamic variable generated for private repositories in the `git-provider-token` key.
-
-For example, to add a comment to a pull request, you can use the `github-add-comment` task from {tekton-hub} using a {pac} annotation:
-
-[source,yaml]
-----
-...
-  pipelinesascode.tekton.dev/task: "github-add-comment"
-...
-----
-
-You can then add a task to the `tasks` section or `finally` tasks in the pipeline run definition:
-
-[source,yaml]
-----
-[...]
-tasks:
-  - name:
-      taskRef:
-        name: github-add-comment
-      params:
-        - name: REQUEST_URL
-          value: "{{ repo_url }}/pull/{{ pull_request_number }}" <1>
-        - name: COMMENT_OR_FILE
-          value: "Pipelines as Code IS GREAT!"
-        - name: GITHUB_TOKEN_SECRET_NAME
-          value: "{{ git_auth_secret }}"
-        - name: GITHUB_TOKEN_SECRET_KEY
-          value: "git-provider-token"
-...
-----
-<1> By using the dynamic variables, you can reuse this snippet template for any pull request from any repository.
-
-[NOTE]
-====
-On GitHub Apps, the generated installation token is available for 8 hours and scoped to the repository from where the events originate unless configured differently on the cluster.
+* For pipelines, use at least one pipeline run with a spec, or a separated `Pipeline` object.
+* For tasks, embed task spec inside a pipeline, or define it separately as a Task object.
```

Comment:
```suggestion
* For tasks, embed the task spec inside a pipeline, or define it separately as a Task object.
```
If there is a specific reason that there isn't an article, then please disregard my comment.